views/layouts/main.handlebars

```diff
@@ -6,41 +6,18 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>One Man's Trash</title>
-  <link rel="stylesheet" href="/css/jass.css">
-  <link rel="stylesheet" href="/css/style.css">
+  <script src="https://kit.fontawesome.com/c1070ced3f.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="/css/style.css"/>
+  
 </head>
 
 <body>
-
-  <div class="flex-column justify-space-around">
-
-    <header class="display-flex justify-space-between align-center p-2">
-
-      <h1>
-        <a href="/">One Man's Trash</a>
-      </h1>
-
-      <nav>
-        <!-- Conditionally render login or logout links -->
-        {{#if logged_in}}
-        <a href="/profile">profile</a> |
-        <button class="no-button" id="logout">logout</button>
-        {{else}}
-        <a href="/login">login</a>
-        {{/if}}
-      </nav>
```

Comment:
Removed this in order to just see the design without anything potentially messing it up. Will put back in but comment it out so code is there but not affecting the view.